testhelpers/testhelpers.go

```diff
@@ -391,3 +399,133 @@ func checkResponseError(r io.Reader) error {
 
 	return nil
 }
+
+func CreateSingleFileTar(path, txt string) (io.Reader, error) {
```

Comment:
```suggestion
func CreateSingleFileTarReader(path, txt string) (io.Reader, error) {
```
This might be better for distinguishing this function from `CreateSingleFileLayerTar`

Reply:
There is already a method with that name `CreateSingleFileTarReader`.  This one is creating a tar file with its header